content/pages/blogposts/2024-10-14-community-over-code-na-birds-of-a-feather.md

```diff
@@ -0,0 +1,11 @@
+Title: Community Over Code NA 2024 Apache Lucene/Solr Birds Of A Feather
+category: solr/blogposts
+summary: This blog post is a summary of the Apache Lucene/Solr Birds of a Feather from Community Over Code NA 2024 in Denver. Written by Eric Pugh and Stefan Vodita.
+slug: community-over-code-na-2024-birds-of-a-feather
+URL: blogposts/community-over-code-na-2024-birds-of-a-feather.html
+save_as: blogposts/community-over-code-na-2024-birds-of-a-feather.html
+
+# Community Over Code NA 2024 Apache Lucene/Solr Birds Of A Feather
+Dive into the insights and key takeaways from the Apache Lucene/Solr Birds of a Feather (BoF) session at Community Over Code EU 2024, held in the mile high city of Denver, Colorado. This session was specially organized to gather feedback from the community, fostering a collaborative environment where developers, users, and enthusiasts could share their experiences and insights. 
+
+Read about the brainstorming that took place during the event to shape the future of these Apache projects - the complete post can be read here: [Community Over Code NA 2024 Apache Lucene/Solr Birds Of A Feather](https://dep4b.hashnode.dev/community-over-code-na-2024-apache-lucenesolr-birds-of-a-feather)
```

Comment:
[Q] In these "external link" blog posts, I'm never sure whether or not to provide feedback on the external post itself, or limit my review to just this little blurb.

But if the "external post" is in scope, a few comments:

> What should we know about Lucene 10?

Putting "Knn Capabilities" in the list that follows the "Lucene 10" lead in implies that Lucene's Knn support is new in 10, when really it's existed since 9.0 or so.

I have similar concerns about the mention of "Binary Partitioning" in a section titled "What's coming up in Lucene in the next 2 years?"  AFAIK binary-partitioning is already available, and doesn't really make sense there...

Reply:
I'll tweak some text in the blog post!